.github/resources/manifests/base/allow-same-namespace-networkpolicy.yaml

```diff
@@ -1,12 +0,0 @@
-apiVersion: networking.k8s.io/v1
```

Comment:
we shouldn't delete these, this is just mimicking what network policy exists in Kubeflow platform deployment